dictionaries/CallMap.php

```diff
@@ -1774,6 +1774,7 @@
 'DateTime::add' => ['static', 'interval'=>'DateInterval'],
 'DateTime::createFromFormat' => ['static|false', 'format'=>'string', 'time'=>'string', 'timezone='=>'?DateTimeZone'],
 'DateTime::createFromImmutable' => ['static', 'datetTimeImmutable'=>'DateTimeImmutable'],
+'DateTime::createFromInterface' => ['DateTimeInterface', 'self'],
```

Comment:
Shouldn't it be in the reverse order? `DateTime::createFromInterface()` returns `self` and accepts `DateTimeInterface`.

Reply:
ouch, thanks! I got confused because first I thought the method was defined in DateTimeInterface and returned static. I fixed the wrong way

Reply:
it's also missing the parameter name, causing 

```
PHP Warning:  Trying to access array offset on value of type int in src/Psalm/Internal/Codebase/InternalCallMapHandler.php on line 266
```

I have submitted #5138 to correct this.